msteams-platform/toolkit/Teams-Toolkit-CLI.md

```diff
@@ -1,23 +1,502 @@
+
+| Command | Description |
+|:----------------  |:-------------|
+| `teamsapp collaborator status` | Show permission status for the project. |
+| `teamsapp collaborator grant` | Check user's permission. |
+
+:::image type="content" source="../assets/images/teams-toolkit-cli/teamsapp-collaborator.png" alt-text="Screenshot shows the teamsapp collaborator commands.":::
+
+### Parameters for `teamsapp collaborator status`
+
+| Parameter | Required | Description |
+|:----------------  |:-------------|:-------------|
+| `--env` | No | Select an existing environment for the project. |
+| `--folder -f` | No | Select root folder of the project. Defaults to `./`. |
+| `--teams-manifest-file -t` | No | Select the manifest template file path, defaults to `${folder}/appPackage/manifest.json`. |
+| `--entra-app-manifest-file -a` | No | Select the Entra manifest template file path, defaults to `${folder}/aad.manifest.json`. |
+| `--all -a` | No | List all collaborators. |
+
+### Parameters for `teamsapp collaborator grant`
+
+| Parameter | Required | Description |
+|:----------------  |:-------------|:-------------|
+| `--env` | Yes | Select an existing environment for the project. |
+| `--folder -f` | No | Select root folder of the project. Defaults to `./`. |
+| `--email` | No | Input email address of the collaborator. |
+| `--teams-manifest-file -t` | No | Select the manifest template file path, defaults to `${folder}/appPackage/manifest.json`. |
+| `--entra-app-manifest-file -a` | No | Select the Entra app manifest file path, defaults to `${folder}/aad.manifest.json`. |
+
+### Scenarios for `teamsapp collaborator`
+
+* Grant permission
+
+  Grant permission for another Microsoft 365 account to collaborate on the Teams app:
+
+  ```typescript
+  teamsapp collaborator grant -i false --teams-manifest-file ./appPackage/manifest.json --env dev --email other@email.com
+  ```
+
+  After you receive the required permission, project creator and collaborators can share the project with the new collaborator by GitHub, and the new collaborator can have all the permissions for Microsoft 365 account.
+
+* Show permission status
+
+  Project creator and collaborators can use `teamsapp collaborator status` command to view Microsoft 365 account permission for specific env:
+
+  ```typescript
+  teamsapp permission status --env dev
+  ```
+
+* List all collaborators
+
+  Project creator and collaborators can use `teamsapp collaborator status` command to view all collaborators for specific env:
+
+  ```typescript
+  teamsapp collaborator status --env dev --list-all-collaborators
+  ```
+
+## `teamsapp update`
+
+Update the Teams app manifest to Teams Developer Portal.
+
+### Parameters for `teamsapp update`
+
+| Parameter | Required | Description |
+|:----------------  |:-------------|:-------------|
+| `--env` | Yes | Select an existing environment for the project. |
```

Comment:
--env is not required
<img width="878" alt="image" src="https://github.com/MicrosoftDocs/msteams-docs/assets/1658418/747b4cba-e537-4098-a70f-74e012f0514c">

Reply:
Updated.